__tests__/newsApi.test.js

```diff
@@ -194,4 +194,67 @@ describe('CORE Task 6: GET request /api/articles/:article_id/comments', () => {
             expect(res.body.msg).toBe('404 Not Found')
            })
   });
+});
+
+describe('CORE Task 7: GET request /api/articles/:article_id/comments', () => {
+  test('Status 201: POST request to send body and add the comment article in the database governed by the article_id requested', () => {
+   const newUser = {
+    username: 'butter_bridge',
+    body: 'I wake up in the morning thinking why do I even bother'
+   }
+    return request(app)
+           .post('/api/articles/1/comments')
+           .send(newUser)
+           .expect(201)
+           .then((res)=>{
+            expect(res.body.comment.author).toBe('butter_bridge')
+            expect(res.body.comment.body).toBe('I wake up in the morning thinking why do I even bother')
+            expect(res.body.comment).toEqual({
+              comment_id: 19,
+              body: 'I wake up in the morning thinking why do I even bother',
+              article_id: 1,
+              author: 'butter_bridge',
+              votes: 0,
+              created_at: expect.any(String)
+            })
```

Comment:
We could add an extra test to check what happens if we have some additional properties, which should send a 201 status and ignore the extra properties 🙂